### PR TITLE
[58] Develop an E-commerce Store Database

### PR DESCRIPTION
**Pull request by kazeemerinfolami**
## Task: `[58] Develop an E-commerce Store Database`
- `Created and E-commerce Store Database`

### CODE REVIEW

File: submissions/kazeemErinfolami/database/init.sql [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+(7, 600.00, now()),
+(1, 390.00, now()),
+(5, 700.00, now()),
+(3, 990.00, now());
+
+SELECT * FROM orders;
+
+
+/*PRODUCTS...*/
+CREATE TABLE products (
```
Comment (adaugochi) [inline submissions/kazeemErinfolami/database/init.sql:127]: Kindly rework this products table again. Some column(s) is/are not supposed to be part of this table

File: submissions/kazeemErinfolami/database/init.sql [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+("Zara shoes", "selection of Loafers, Formal Shoes, Boots, Sneakers & more", "../html/images/brogues_shoes.jpg", 490.00, 6, 'out of stock', 1, 3, 4 ),
+("lip stick", "selection of Loafers, Formal Shoes, Boots, Sneakers & more", "../html/images/women acc.jpg", 50.00, 6, 'in of stock', 3, 5, 3 ),
+("LG", "feel free and code.", "../html/images/laptop.jpg", 190.00, 3, 'in stock', 5, 1, 2 ),
+("lip stick", "selection of Loafers, Formal Shoes, Boots, Sneakers & more", "../html/images/women acc.jpg", 50.00, 6, 'in of stock', 3, 8, 1 );
+
+SELECT * FROM products;
+
+
+/*ORDERS ITEMS...*/
+CREATE TABLE orders_items (
```
Comment (adaugochi) [inline submissions/kazeemErinfolami/database/init.sql:173]: Kindly rework this order_items table again. Some column(s) is/are not supposed to be part of this table

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/kazeemErinfolami/database/init.sql [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+("lip stick", "selection of Loafers, Formal Shoes, Boots, Sneakers & more", "../html/images/women acc.jpg", 50.00, 6, 'in of stock', 3, 1 );
+
+SELECT * FROM products;
+
+
+/*ORDERS ITEMS...*/
+CREATE TABLE orders_items (
+	id INT NOT NULL AUTO_INCREMENT,
+    order_id INT NOT NULL,
+    admin_id INT NOT NULL,
```
Comment (adaugochi) [inline submissions/kazeemErinfolami/database/init.sql:173]: An order item cannot have `admin_id`. There is still one missing column on this table. Kindly read through the task again. Well done @kazeemerinfolami

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by kazeemerinfolami** [comment]
I have made the changes. Thank you

**Review by adaugochi — APPROVED** [review]

**Pull request merged**